pkg/interface/src/views/components/VirtualScroller.tsx

```diff
@@ -575,8 +574,10 @@ function VirtualChild(props: VirtualChildProps) {
 
   const ref = useCallback((el: HTMLElement | null) => {
     setRef(el, props.index);
-  }, [setRef, props.index])
+  //  VirtualChild should always be keyed on the index, so the index should be
+  //  valid for the entire lifecycle of the component, hence no dependencies
+  }, []);
```

Comment:
I believe this makes it update on every tick

Reply:
No, that's if the dependencies are not provided. `[]` will cause the ref callback to be constant throughout the entire lifecycle.